src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -69,6 +69,7 @@
     <Compile Include="Serialization\PropertyNameTests.cs" />
     <Compile Include="Serialization\PropertyVisibilityTests.cs" />
     <Compile Include="Serialization\ReadValueTests.cs" />
+    <Compile Include="Serialization\SnakeCaseUnitTests.cs" />
```

Comment:
It will be good to bring in some of the tests from this PR as well:
https://github.com/dotnet/corefx/pull/40003